### ISSUE TITLE
Multiple Issues with communicator app

### ISSUE THREAD

**Issue by georgeshaine** [OfficeDev/microsoft-teams-apps-company-communicator#218]
Hello,
As per recommendation of Madhan Kumar ,  raising the issue i have been facing with deployment on my site , the  deployment is completed as per the deployment guide but then the app has many many errors as detailed below.
Nothing shows up in picker for send to specific teams and send to specific people.
Preview in this channel throws an error.
Send to M365 / distribution group / security groups throws an error when trying to search for a group. 
If exact group is copied and pasted and message attempted to send, it fails.
Export of the detailed error also fails

**Comment by georgeshaine** [comment]
update
checked AAD app registration, verified API resources set correctly , verified graph API permissions are set, client secret is correct, sign in url is correct. In teams app manifest the single sign on is set correctly. This is a European tenant.

**Comment by DebGwynne** [comment]
The send to M365 error has been resolved in the latest version of the app, I had that, and syncing down the latest version resolved it.

The export of the detailed error also failed for us because the permissions were incorrectly set to application rather than delegate (or vice versa, I didn't do the Azure config, was just helpfully talked through correcting it).  So that might be worth a check.

I initially couldn't find teams to send to, and that was because our global permission policy was set to not allow interaction with custom apps, so I had to add the author app in, iirc.

Note:  Not a dev, but sounds like some similar issues to us so maybe some of that will help!
